server/webapp/WEB-INF/rails.new/app/assets/stylesheets/css/build_detail.scss

```diff
@@ -290,22 +290,63 @@ div.build_detail_container_content {
 .widget iframe {
     border: none;
 }
+
+#console-action-bar {
+    min-height: 20px;
+    padding: 10px;
+
+    background-color: #333;
+
+    div {
+        float: right;
+    }
+
+    a {
+        padding-right: 15px;
+        color: yellow;
+    }
+
+    &.white-theme {
+        background-color: white;
+        a {
+            color: #1F66BD;
```

Comment:
Ha ha. Change it: [https://github.com/gocd/gocd/blob/master/.hound.yml](https://github.com/gocd/gocd/blob/master/.hound.yml). I don't think case should matter. I suppose it's the default so that it will be consistent.